src/Microsoft.ML.LightGbm/LightGbmArguments.cs

```diff
@@ -58,6 +59,11 @@ public BoosterParameterBase(OptionsBase options)
         public abstract class OptionsBase : IBoosterParameterFactory
         {
             internal BoosterParameterBase GetBooster() { return null; }
+            /// <summary>
+            /// Whether training data is unbalanced. Used by <see cref="LightGbmBinaryTrainer"/>.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, HelpText = "Use for binary classification when training data is not balanced.", ShortName = "us")]
+            public bool UnbalancedSets = false;
```

Comment:
Where is this boolean being used? I don't see any code added using it.

Reply:
It's added to the dictionary, NameMapping on line 39:
{nameof(OptionsBase.UnbalancedSets),                 "is_unbalance"},

Reply:
You are declaring 2 new boolean fields. I only see 1 being used.